### ISSUE TITLE
zcl-electrum.com server not connecting

### ISSUE THREAD

**Issue by ghost** [z-classic/electrum-zcl#60]
Hi
I downloaded the latest version of the wallet, and its still not connecting since morning. the server on the list is only zcl-electrum.com port 50002

is there any other?

thanks

**Comment by FrankAllotta** [comment]
Same.  Wallet just says Synchronizing.  Never Synchronizes.

**Comment by ghost** [comment]
the problem is I've moved my coins to it since it was working fine the day before, then it just stopped connecting

**Comment by FrankAllotta** [comment]
I hear ya.  I haven't been able to find a working Zclassic wallet that is reliable.  I hope this thing is working come fork time.  There are some potentially disastrous financial consequences if it isn't.  If I transfer all my ZCL on the exchange to a wallet that doesn't work, then the fork occurs and ZCL goes to zero, I won't have any way of getting in on the selloff.  Not to mention, does ZCL just get abandoned after the fork?  It's going to be interesting.

**Comment by ghost** [comment]
the elios wallet was okay really not sure why i transferred to this, should have stayed on elios, can't even transfer it back there, lets hope its resolved soon

**Comment by ghost** [comment]
I just tried sending funds from the wallet to my other and it worked, but its still not synchronising

**Comment by ch4ot1c** [comment]
Please try this latest release, and post again if you are still having issues - https://github.com/BTCP-community/electrum-zcl/releases/tag/Z!1.0.3

**Comment by FrankAllotta** [comment]
Works like a champ Jon @ch4ot1c!  I can see my balance from a shielded transaction sender now. You guys are doing great work.  Keep it up.

**Comment by ghost** [comment]
works great, thank you so much

**Comment by awllc** [comment]
I have only used that latest release, and am having the exact same problem as the OP.

First it wouldn't let me back into the wallet I first made, even though I used the exact same password. So I deleted the folder and recreated the wallet. 

But has never sync'd. Doesn't even say it is trying to sync. I have the same zcl-electrum.com option under host and port as the OP, but there is no button to initiate sync (only 'close') and it does not automatically sync.
